explainer.md

```diff
@@ -36,10 +36,13 @@ For an [example of how to use the API](https://w3c-fedid.github.io/digital-crede
 
 ### Using the API from another origin
 
-The specification allows usage of the API from a remote/third-party origin via the "digital-credentials-get" Permissions Policy. This is useful for scenarios where a website wants to request digital credentials from a wallet provider that is hosted on a different origin. The Permissions Policy can be set on an iframe that embeds the website that wants to use the API. Here is an example of how the Permissions Policy can be set on an iframe:
+
```

Comment:
FYI... I think we should remove this from the Explainer:
https://github.com/w3c-fedid/digital-credentials/pull/322

Having to maintain two documents that same the same thing is no fun.